### PR TITLE
Add stream8 buildroot supplemental CI image

### PR DESCRIPTION
**Pull request by osherdp**
/hold
/cc @YuviGold @celebdor

### CODE REVIEW

File: clusters/app.ci/supplemental-ci-images/coreos/coreos-stream8-buildroot/build-base.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -xeuo pipefail
+# First, fix coreutils
+yum -y swap coreutils{-single,}
```
Comment (flaper87) [inline clusters/app.ci/supplemental-ci-images/coreos/coreos-stream8-buildroot/build-base.sh:4]: nit: I think we can use `dnf` instead of `yum` here

**Review by flaper87 — COMMENTED** [review]
Looks good! I will let someone else approve it, tho :)

File: clusters/app.ci/supplemental-ci-images/coreos/coreos-stream8-buildroot/build-base.sh (hunk 1 above)
Comment (celebdor) [reply clusters/app.ci/supplemental-ci-images/coreos/coreos-stream8-buildroot/build-base.sh:4]: There's an alias thankfully. We  can move it to dnf/microdnf later

Comment (flaper87) [reply clusters/app.ci/supplemental-ci-images/coreos/coreos-stream8-buildroot/build-base.sh:4]: > There's an alias thankfully. We can move it to dnf/microdnf later

I know it's an alias but all these files are new, we could just `search and replace` :smiley: 

(also, I labeled my comment as `nitpick` so, not a blocker!)

**Review by stevekuznetsov — APPROVED** [review]

**Review by djzager — APPROVED** [review]
/hold cancel

**Pull request merged**